servers/rendering_server.h

```diff
@@ -1260,6 +1260,12 @@ class RenderingServer : public Object {
 		SHADOW_CASTING_SETTING_SHADOWS_ONLY,
 	};
 
+	enum ShadowDynamicMode {
```

Comment:
This probably should be a more general hint whether the object is going to move, maybe they should be merged somehow with others.

Reply:
I'm open for ideas here, totally agree that having different settings that are similar/same doesn't make sense however many of the existing systems can't do automatic as this requires a running project.